Extension/src/LanguageServer/client.ts

```diff
@@ -2171,7 +2166,7 @@ export class DefaultClient implements Client {
 
     public getVcpkgEnabled(): Promise<boolean> {
         const cppSettings: CppSettings = new CppSettings(this.RootUri);
-        return Promise.resolve(!!cppSettings.vcpkgEnabled);
+        return Promise.resolve(!!cppSettings.isVcpkgEnabled);
```

Comment:
You should be able to remove the `!!` here now.

Reply:
Why are we using the !! syntax? Just asking this to learn.

Reply:
`!!` is a way to convert non-booleans to a boolean.  Many types can be treated as if they were boolean, but they could be other things like `number` or `string`.  This syntax makes sure that we return an actual boolean instead of something else.

I don't remember the history for this particular case because it looks like the setting should have been a boolean already, but I know for sure that it's not anymore so the `!!` can be removed.